src/core/Akka.Persistence.Tests/Serialization/MessageSerializerRemotingSpec.cs

```diff
@@ -23,12 +23,12 @@ internal class Configs
 akka.actor {
   serializers {
     my-payload = ""Akka.Persistence.Tests.Serialization.MyPayloadSerializer, Akka.Persistence.Tests""
-    my-payload2 = ""Akka.Persistence.Tests.Serialization.MyPayload2Serializer, Akka.Persistence.Tests""
     old-payload = ""Akka.Persistence.Tests.Serialization.OldPayloadSerializer, Akka.Persistence.Tests""
+    testserializer = ""Akka.Serialization.HyperionSerializer, Akka.Serialization.Hyperion""
```

Comment:
In order to fix these tests. I switched system default serializer for these tests to Hyperion.
But im not sure if thats a good idea.